### PR TITLE
suggested offer of help

### PR DESCRIPTION
**Pull request by urbanjost**
Not sure if everyone is open to just suggesting someone open an issue and ask for help but throwing it out there
for consideration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,22 +7,21 @@ packages for the [fortran-lang](https://fortran-lang.org) site.
...
 Your new registry submittal should first meet the 
 [general package criteria](https://github.com/fortran-lang/fortran-lang.org/blob/master/PACKAGES.md)
-required of any package 
-listed at the [packages listing](https://fortran-lang.org/packages).
+required of any package listed at the
+[packages listing](https://fortran-lang.org/packages).
 
-Please submit a pull request against this repository, add the new package into the file
+Please submit a **PR** or
+[pull request](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```
Comment (certik) [inline README.md]:
```suggestion
Please submit a Pull Request ([PR](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github))
```

File: README.md [hunk 2]
```diff
@@ -47,3 +46,14 @@ python load_registry.py
...
 When a pull request is merged it will then appear at the
 [`fpm registry`](https://fortran-lang.org/packages/fpm).
+
+## Are you kidding?
+
+If you have a great Fortran package you have managed to get onto
+**github** or another **git** repository that you are interested
+in setting up as a registered **fpm** package that adheres to the
+guidelines listed here but terms like **fpm**, **PR**, **toml**, and
+**python** are putting you off from registering your package open an
```
Comment (certik) [inline README.md]:
```suggestion
**python** are putting you off from registering your package, please open an
```

File: README.md [hunk 3]
```diff
@@ -47,3 +46,14 @@ python load_registry.py
...
+[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
+**fpm Registry** site and list your Repository and ask for help.  Get as
+far as you can and identify where you got stuck.
```
Comment (certik) [inline README.md]:
```suggestion
far as you can and identify where you got stuck. We will try to help.
```

File: README.md [hunk 4]
```diff
@@ -47,3 +46,14 @@ python load_registry.py
 
 When a pull request is merged it will then appear at the
 [`fpm registry`](https://fortran-lang.org/packages/fpm).
+
+## Are you kidding?
```
Comment (certik) [inline README.md]:
```suggestion
## How to request help
```

**Review by certik — CHANGES_REQUESTED** [review]
I think that's a good idea. I left some comments.

**Comment by urbanjost** [comment]
Thanks.  Basically the request is that i"f you have a Fortran project in a git(1) repository you would like to share and have converted to an fpm(1) package please submit its' name for review".  There are quite a few steps being required 
to do that that I believe are off-putting for a good number of Fortran programmers. Seeing how many people ask for help might prove me wrong, but I am glad so see support for offering the time to help people over that threshold.

File: README.md [hunk 5]
```diff
@@ -47,3 +45,14 @@ python load_registry.py
...
+## How to request help
+
+If you have a great Fortran package you have managed to get onto
+**github** or another **git** repository that you are interested
+in setting up as a registered **fpm** package that adheres to the
+guidelines listed here but terms like **fpm**, **PR**, **toml**, and
+**python** are putting you off from registering your package, please open an
+[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
+**fpm Registry** site and list your Repository and ask for help.  Get as
+far as you can and identify where you got stuck. We will try to help.
```
Comment (LKedward) [inline README.md]:
```suggestion
If you have a great Fortran package that you are interested
in registering as an **fpm** package but terms like **fpm**, **PR**, **toml**, and
**python** are putting you off, then please open an
[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
**fpm Registry** site listing your Repository and let us know how we can help. Get as
far as you can and identify where you got stuck. We are always happy to help.
```

This is a *very* long multi-clause sentence; I have removed some redundancy and simplified.

File: README.md [hunk 6]
```diff
@@ -7,22 +7,20 @@ packages for the [fortran-lang](https://fortran-lang.org) site.
...
+required of any package listed at the
+[packages listing](https://fortran-lang.org/packages).
 
-Please submit a pull request against this repository, add the new package into the file
+Please submit a Pull Request ([PR](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github))
+against this repository, adding the new package into the file
 [registry.toml](./registry.toml)
-in alphabetical order. Explicitly list each
-version using the 
+in alphabetical order. Explicitly list each version using the
```
Comment (LKedward) [inline README.md]:
```suggestion
in alphabetical order. It is recommended that you explicitly list each version using the 
```

File: README.md [hunk 7]
```diff
@@ -7,22 +7,20 @@ packages for the [fortran-lang](https://fortran-lang.org) site.
...
+Please submit a Pull Request ([PR](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github))
+against this repository, adding the new package into the file
 [registry.toml](./registry.toml)
-in alphabetical order. Explicitly list each
-version using the 
+in alphabetical order. Explicitly list each version using the 
 [`tag`](https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/managing-tags)
-keyword, otherwise the master branch will be used. An
-example:
+keyword, otherwise the master branch will be used. An example:
```
Comment (LKedward) [inline README.md]:
```suggestion
keyword, otherwise the tip of the master branch will be used. An example:
```

**Review by LKedward — APPROVED** [review]
Thanks @urbanjost, it's good to remove as many barriers as possible to new contributors.

**Comment by certik** [comment]
The way it will work is that right now we are happy to invest our time to help anyone who asks. As we get a bigger community, there will be more people to help, so it will scale. Just like in other communities, you just ask for help and somebody will help.

Why don't we also point them to our Discourse forum?

File: README.md [hunk 8]
```diff
@@ -7,22 +7,20 @@ packages for the [fortran-lang](https://fortran-lang.org) site.
...
 [`tag`](https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/managing-tags)
-keyword, otherwise the master branch will be used. An
-example:
+keyword, otherwise the tip of the master branch will be used. An example:
 
 ```toml
 [datetime]
 "1.7.0" = {git="https://github.com/wavebitscientific/datetime-fortran", tag="v1.7.0"}
 "latest" = {git="https://github.com/wavebitscientific/datetime-fortran"}
-```
```
Comment (awvwgk) [inline README.md:25]: This line was probably deleted by accident

**Comment by urbanjost** [comment]
I tend to do that on a draft and then catch it. I remember Miss Dore (my English teacher) reminding me there is a difference between a paragraph and a sentence.  I tend to backslide when I'm in a hurry, I'm afraid.

**Review by certik — APPROVED** [review]
This looks good to me now. Thanks!

File: registry.toml [hunk 9]
```diff
@@ -38,18 +38,18 @@
...
+"latest" = {git="https://github.com/urbanjost/M_pixel"}
+
 [M_process]
 "latest" = {git="https://github.com/urbanjost/M_process"}
 
+[M_system]
+"latest" = {git="https://github.com/urbanjost/M_system"}
+
 [M_strings]
 "latest" = {git="https://github.com/urbanjost/M_strings"}
```
Comment (milancurcic) [inline registry.toml]: Alphabetical order:

```suggestion
[M_strings]
￼"latest" = {git="https://github.com/urbanjost/M_strings"}

[M_system]
￼"latest" = {git="https://github.com/urbanjost/M_system"}￼
```

**Review by milancurcic — APPROVED** [review]
Looks good. In the future, let's handle package submissions to the registry in a dedicated PR.

**Pull request merged**